src/main/kotlin/com/sourcegraph/cody/history/state/MessageState.kt

```diff
@@ -0,0 +1,27 @@
+package com.sourcegraph.cody.history.state
+
+import com.intellij.openapi.components.BaseState
+import com.sourcegraph.cody.agent.protocol.Speaker
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class MessageState : BaseState() {
+
+  var text: String? by string()
```

Comment:
Do I understand correctly, that passing `null` there causes issues during deserialisation?
If so, why do we allow data in model to be `null` in the first place?